frontend/public/components/factory/list.tsx

```diff
@@ -13,9 +13,10 @@ import { getJobTypeAndCompletions, isNodeReady, podPhase, podPhaseFilterReducer,
 import { isScanned, isSupported, makePodvuln, numFixables } from '../../module/k8s/podvulns';
 import { UIActions } from '../../ui/ui-actions';
 import { ingressValidHosts } from '../ingress';
+import { routeStatus } from '../routes';
+import { secretTypeFilterReducer } from '../secret';
 import { bindingType, roleType } from '../RBAC';
 import { LabelList, ResourceCog, ResourceLink, resourcePath, Selector, StatusBox, containerLinuxUpdateOperator, EmptyBox } from '../utils';
-import { routeStatus } from '../routes';
```

Comment:
Not sure if this change is necessary ?

Reply:
Not necessary, but I wanted to keep similar imports together. We might look at how we order imports in general. They're kind of a mess :/